node/testing.go

```diff
@@ -191,7 +191,11 @@ func MakeOfflineNode(t *testing.T) *Node {
 // DefaultTestingConfig returns default configuration for testing
 func DefaultTestingConfig() []ConfigOpt {
 	return []ConfigOpt{
-		VerifierConfigOption(verification.NewFakeVerifier(true, nil)),
+		VerifierConfigOption(verification.NewFakeVerifier(verification.FakeVerifierConfig{
```

Comment:
> Is this necessary?

It may not be necessary, but it's also not something we introduced. Note the line which we replaced:

```
VerifierConfigOption(verification.NewFakeVerifier(true, nil)), 
```

In order to limit the chance of regressing the application, we simply retrofitted the old config to conform to the new API.